certified-connectors/ascentcloudterritoryplanner/apiDefinition.swagger.json

```diff
@@ -0,0 +1,235 @@
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Territory Plan",
+        "description": "Get Territory Plan",
+        "operationId": "public_v1_get_territory_plan",
+        "parameters": [
+          {
+            "name": "territory_plan_id",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Territory Plan Id",
+            "x-ms-summary": "Territory Plan Id"
+          },
+          {
+            "name": "map_object",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Map Object",
+            "x-ms-summary": "Map Object"
+          },
+          {
+            "name": "x-org-id",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Organization Id",
+            "x-ms-summary": "Territory Planner Organization Id"
+          }
+        ]
+      }
+    },
+    "/territory_data_point_assignments": {
+      "get": {
+        "responses": {
+          "400": {
+            "description": "An Error Occurred"
+          },
+          "401": {
+            "description": "Unauthorized"
+          },
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "data": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "description": "id"
+                      },
+                      "externalId": {
+                        "type": "string",
+                        "description": "externalId"
+                      },
+                      "territoryId": {
+                        "type": "string",
+                        "description": "territoryId"
+                      },
+                      "locked": {
+                        "type": "boolean",
+                        "description": "locked"
+                      },
+                      "latitude": {
+                        "type": "number",
+                        "format": "float",
+                        "description": "latitude"
+                      },
+                      "longitude": {
+                        "type": "number",
+                        "format": "float",
+                        "description": "longitude"
+                      }
+                    }
+                  },
+                  "description": "data"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Territory Data Point Assignments",
+        "description": "Get Territory Data Point Assignments",
+        "operationId": "public_v1_get_territory_data_point_assignments",
```

Comment:
@simongroom all operationIds should be in Camel case. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).